### ISSUE TITLE
Picker jump

### ISSUE THREAD

**Issue by Dani79** [amsul/pickadate.js#357]
The picker jump, there is a scrollbar for a few mseconds. The solution:

.picker__holder {overflow-y:hidden;}

hope this is right?!

**Comment by amsul** [comment]
Yep - it's a bit of a tough situation to get around because of the transitions. I'll keep this around if anyone can come up with an elegant solution to it.
